src/plugins/intel_cpu/src/nodes/eltwise.cpp

```diff
@@ -2138,40 +2138,44 @@ void Eltwise::appendPostOpsImpl(mkldnn::post_ops& ops, const VectorDims &postOpD
         default: IE_THROW() << errorPrefix << "as post operation is not supported";
         }
     } else {
-        const size_t chIdx = postOpDims.size() > 1 ? getFusingAxis() : 0;
+        size_t channelSize = 1;
+        if (static_cast<int>(channelAxis) >= 0) {
```

Comment:
`size_t` cast to `int` is not good, if channelAix is size_t, it should be positive. If it has negative value, why not using int for type ?

Reply:
Fixed. Thanks Yi!